packages/amplify_core/lib/src/types/analytics/analytics/user_profile_location.dart

```diff
@@ -1,15 +1,15 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-class AnalyticsUserProfileLocation {
+class UserProfileLocation {
```

Comment:
Ditto

Reply:
What's the plan communicate the migration plan? i.e. `remove "Analytics" prefix`?

Also `customEvent.properties` => `customEvent.customProperties`.

Reply:
Yes I will need to update the documentation.  My understanding is that Dillon will change docs to allow v1 vs. v2 docs.  Once that is merged I will add a docs PR to include a warning for these breaking changes.